src/index.html

```diff
@@ -17,16 +17,26 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input
+          data-qa="keypress"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          class="search-form__input search-form__input_big"
+        />
+      </label>
+    </form>
```

Comment:
There should be an empty line between multiline sibling blocks of HTML. Please add an empty line between these two form elements. [HTML Formatting]